.github/workflows/integration.yml

```diff
@@ -271,6 +271,26 @@ jobs:
       - name: build proposals tests
         run: yarn build
         working-directory: a3p-integration
+      - name: zip up all core proposals
+        run: |
+          find . -type d -path "./a3p-integration/proposals/*/submission" | while read -r dir; do
+            parent_dir=$(basename "$(dirname "$dir")")
+
+            sanitized_parent_dir=${parent_dir//:/-}
+
+            zip_file="${sanitized_parent_dir}.zip"
+
+            zip -r "$zip_file" "$dir"
+
+            echo "Zipped $dir into $zip_file"
+          done
+      - name: archive core proposals
+        uses: actions/upload-artifact@v4
+        with:
+          name: core-proposals
+          path: ./*.zip
```

Comment:
why zip instead of uploading the dirs?
```
    path: ./a3p-integration/proposals/*/submission
```

Reply:
Because `upload-artifact` action doesn't like it when there's `:` in the path, we have `b:upgrade-next` in the path which would prevent the action from uploading. So I opted to zip while replacing the `:` with `-` instead of renaming the directories

Reply:
Ah. Please document that in a comment on the zipping step

Reply:
Alternately, it would be less code and perhaps a better debugging experience if instead of zipping the submissions were copied to a /tmp folder and this step uploaded from there. (copy with rename of parent path)